src/Fhir.Anonymizer.Shared.Core/AnonymizationConfigurations/AnonymizationFhirPathRule.cs

```diff
@@ -67,6 +81,8 @@ public AnonymizationFhirPathRule(string path, string expression, string resource
 
             Expression = expression;
             ResourceType = resourceType;
+            ReplaceWith = replaceWith;
```

Comment:
Let's not do parse for specific processor.

Reply:
Thanks. Fixed